command/whitelist/whitelist.go

```diff
@@ -0,0 +1,25 @@
+package whitelist
+
+import (
+	"github.com/0xPolygon/polygon-edge/command/whitelist/deployment"
+	"github.com/0xPolygon/polygon-edge/command/whitelist/show"
+	"github.com/spf13/cobra"
+)
+
+func GetCommand() *cobra.Command {
+	whitelistCmd := &cobra.Command{
+		Use:   "whitelist",
+		Short: "Top level command for modifying the Polygon Edge whitelists within config file. Only accepts subcommands.",
```

Comment:
Typo here, should be `within the`

Reply:
fixed 0cf7fa2192ad0eba12eda7d8e57efacd9330b98f

Reply:
Lol this created linter error (lll)

Reply:
Fixed linter 2fe0620c2994d5d94244529cfa57ad28197ff41f